test/TesterInternal/ErrorInjectionStorageProvider.cs

```diff
@@ -53,6 +48,14 @@ public override string Message
                 return "ErrorInjectionPoint=" + Enum.GetName(typeof(ErrorInjectionPoint), errorInjectionPoint);
             }
         }
+
+#if !NETSTANDARD
```

Comment:
Please undo this change after rebasing on top of `master`, since we removed all NETSTANDARD conditional compilation statements in #3368 that was just merged